src/renderer/entities/wallet/lib/__tests__/import-keys-utils.test.ts

```diff
@@ -0,0 +1,105 @@
+import { KeyType } from '@renderer/shared/core';
+import { importKeysUtils } from '../import-keys-utils';
+import { ImportedDerivation } from '../types';
+import { importKeysMocks } from './mocks/import-keys-utils.mock';
+
+describe('entities/dynamicDerivations/import-keys-utils', () => {
+  describe('entities/dynamicDerivations/import-keys-utils/validateDerivation', () => {
+    test.each([
+      [importKeysMocks.invalidDerivations.passwordPath, false],
+      [importKeysMocks.invalidDerivations.invalidPath, false],
+      [importKeysMocks.invalidDerivations.emptyPath, false],
+      [importKeysMocks.invalidDerivations.tooManyShards, false],
```

Comment:
Sorry, the requirement has been changed. Now the minimum number of shards has to be checked too. The minimum number of shards is 2.